example/ck_tile/03_gemm/script/benchmark_mem_pipeline.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+EXE="$(find . -name tile_example_gemm_universal -type f | head -n 1)"
+VALID=0
+
+for b in "R" "C"; do
```

Comment:
What is b? Can we use some better shortcut? (here and in other files)

Reply:
Thank you! Comment Resolved.